input/core/src/commonMain/kotlin/majestic/editor/ToolBarHost.kt

```diff
@@ -46,26 +50,30 @@ fun Modifier.borderBottom(
 
 @Composable
 fun ToolBarHost(
+    modifier: Modifier,
```

Comment:
Since ToolBarHost exists in majestic (a library), give the `modifier` a default value, so that the caller, doesn't have to specify if they don't need it.


And since this property will have a default value, it shouldn't be defined before parameters with default values